src/pytest_inmanta_lsm/resources/setup_project.py

```diff
@@ -76,10 +96,15 @@
     v2_modules.append(mod)
     LOGGER.info(f"Module {mod.name} is v2, we will attempt to install it")
 
-# Install all v2 modules in editable mode
+# Install all v2 modules in editable mode using the project's configured package sources
 if v2_modules:
-    LOGGER.info(f"Installing modules from source: {[mod.name for mod in v2_modules]}")
-    project.virtualenv.install_from_source([env.LocalPackagePath(mod.path, editable=True) for mod in v2_modules])
+    urls: abc.Sequence[str] = project.module_source.urls
+    if not urls:
+        raise Exception("No package repos configured for project")
+    # plain Python install so core does not apply project's sources -> we need to configure pip index ourselves
+    with env_vars({"PIP_INDEX_URL": urls[0], "PIP_EXTRA_INDEX_URL": " ".join(urls[1:])}):
```

Comment:
This is not going to work because of: https://github.com/inmanta/inmanta-core/pull/4776

Reply:
I thought that for a moment but that PR should really only change the behavior of `install_from_index` I think. `install_from_source` doesn't have an index argument so it should just use the environment, shouldn't it?

Reply:
(I'll comment it there and see if I need changes here once we've reached a concensus.)